src/commcare_cloud/ansible/roles/nodejs/tasks/main.yml

```diff
@@ -48,6 +48,6 @@
   npm: name="{{ item.name }}" version="{{ item.version }}" state=present global=yes
   become: yes
   with_items:
-    - {name: 'npm', version: '6.14.5'}
+    - {name: 'npm', version: '7.24.2'}
```

Comment:
I see npm is updated both here and in commcare-hq `package.json`/`yarn.lock`. Is that because this manages the global npm install and that (`package.json`) manages a local `npm` install, and we need both? Curious if you know what we need each for

Reply:
yes, npm is needed for the `yarn` install and we also ask `yarn` to build packages from certain `npm` repos